proto/src/main/proto/Rss.proto

```diff
@@ -662,4 +669,30 @@ message ReassignOnBlockSendFailureResponse {
   MutableShuffleHandleInfo handle = 3;
 }
 
+message StartSortMergeRequest {
+  string appId = 1;
+  int32 shuffleId = 2;
+  int32 partitionId = 3;
+  bytes uniqueBlocksBitmap = 4;
+}
+
+message StartSortMergeResponse {
+  StatusCode status = 1;
+  string retMsg = 2;
+}
+
+message GetSortedShuffleDataRequest {
+  string appId = 1;
+  int32 shuffleId = 2;
+  int32 partitionId = 3;
+  int64 blockId = 4;
+  int64 timestamp = 5;
```

Comment:
Why do we need timestamp?

Reply:
Same with GetLocalShuffleDataRequest::timestamp and GetMemoryShuffleDataRequest::timestamp. It may be a cost rpc, so record the time.